transport/example/config_auth.json

```diff
@@ -0,0 +1,37 @@
+{
+    "auth_keys": {
+        "client": {
+            "type": "standard",
+            "key": "52abb3ac91971bb72bce17e7a289cd04476490b19e0d8eb7810dc42d4ac16c41"
```

Comment:
For the example, they should be. The goal is to provide everything to the user to run the examples (except the certificate and key, which are generated with the provided script).